libs/dh/admin/feature-permissions-new/src/lib/table.component.ts

```diff
@@ -58,58 +48,40 @@ import { DhPermissionRequiredDirective } from '@energinet-datahub/dh/shared/feat
   imports: [
     TranslocoPipe,
     TranslocoDirective,
+    RouterOutlet,
 
     WATT_CARD,
     WATT_TABLE,
-    WattSearchComponent,
-    WattButtonComponent,
-    WattEmptyStateComponent,
+    WattDataTableComponent,
+    WattDataActionsComponent,
 
-    VaterFlexComponent,
-    VaterStackComponent,
-    VaterSpacerComponent,
     VaterUtilityDirective,
 
-    DhPermissionRequiredDirective,
+    DhPermissionsDownloadComponent,
   ],
-  template: `<watt-card
-    vater
-    inset="ml"
-    *transloco="let t; read: 'admin.userManagement.permissionsTab'"
-  >
-    <vater-flex fill="vertical" gap="m">
-      <vater-stack direction="row" gap="s">
+  providers: [DhNavigationService],
+  template: `
+    <watt-card vater inset="ml" *transloco="let t; read: 'admin.userManagement.permissionsTab'">
+      <watt-data-table
+        vater
```

Comment:
issue: the page looks like there are nested cards. The `watt-card` should probably be removed?

![image](https://github.com/user-attachments/assets/7596bbe9-979a-47ed-bc61-db67eccbd39c)

Reply:
You are right, data table has its own one